src/workerd/api/streams/internal.c++

```diff
@@ -96,23 +94,24 @@ private:
   kj::Vector<kj::Array<kj::byte>> parts;
   uint64_t runningTotal = 0;
 
-  kj::Promise<PartList> loop() {
-    auto bytes = kj::heapArray<kj::byte>(4096);
+  kj::Promise<PartList> readParts() {
+    auto buffer = kj::heapArray<kj::byte>(4096);
```

Comment:
Similar to @kentonv's comment in the other PR, can this be stack allocated instead?